src/rpc/handlers/AMMInfo.cpp

```diff
@@ -0,0 +1,225 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of clio: https://github.com/XRPLF/clio
+    Copyright (c) 2023, the clio developers.
+
+    Permission to use, copy, modify, and distribute this software for any
+    purpose with or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL,  DIRECT,  INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#include <data/DBHelpers.h>
+#include <rpc/AMMHelpers.h>
+#include <rpc/handlers/AMMInfo.h>
+
+#include <ripple/protocol/AMMCore.h>
+
+namespace {
+
+std::string
+toIso8601(ripple::NetClock::time_point tp)
```

Comment:
Can be replaced by **to_string_iso** ? u can find its usage in our codebase.

Reply:
I wonder why they made a new method 

https://github.com/XRPLF/rippled/blob/8ce85a9750568b8f41f24e65e6103cc256a4f5b0/src/ripple/rpc/handlers/AMMInfo.cpp#L62-L71

From what I understand to_string_iso is more general. But I need another pair of eyes to look into this... I am not 100% sure yet

Reply:
Actually it looks like to_string_iso is new code https://github.com/XRPLF/rippled/commit/32ced493de22601b70234ba215d11e3666cc14cb

Reply:
Feels like it should be safe to replace...

Reply:
Looks like the format is different. Let us keep this local implementation for amm_info for now to match rippled.

Reply:
Okay, thanks for looking into it cindy.. I'm gonna resolve this convo for now